docs/guides/main.rst

```diff
@@ -46,6 +45,8 @@ Beets works on `Python 2.7`_ and Python 3.4 or later.
 
 * On **NixOS**, run ``nix-env -i beets``.
 
+* On most versions of **Windows**, there is a `installer`_ available on the Python website. 
```

Comment:
In this list, we're pointing toward places where you can install *beets* from the package manager, not just Python.